src/visitor/mssql.rs

```diff
@@ -620,6 +620,14 @@ impl<'a> Visitor<'a> for Mssql<'a> {
         unimplemented!("Full-text search is not yet supported on MSSQL")
     }
 
+    #[cfg(feature = "postgresql")]
```

Comment:
This looks off, why is there a postgres feature flag in mssql?

Reply:
This is needed for when we have `features=all`